src/common/module_builder.cc

```diff
@@ -96,11 +96,15 @@ void ModuleBuilder::BuildModule(std::string_view code, std::string_view format,
   {
       throw std::runtime_error("Failed to run MLIR compiler pass pipeline.");
   }
-  DLOG_F(LOG_DEBUG, "TTIR Module");
+  shlo_pm.addPass(mlir::tt::ttir::createTTIRGatherPatternMatch());
```

Comment:
Instead of adding pass manually, can you just invoke the pipeline, like we do for TTIRToTTNN below?